doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -539,6 +539,112 @@ Requesting information
    fixpoint and cofixpoint is violated at some time of the construction
    of the proof without having to wait the completion of the proof.
 
+.. _showing_diffs:
+
+Showing differences between proof steps
+---------------------------------------
+
+
+Coq can automatically highlight the differences between successive proof steps.
+For example, the following screen shots of coqtop and CoqIDE show the same
+“intros” tactic.  This tactic creates two new hypotheses, highlighted in green.
+The conclusion is entirely in pale green because it’s changed, but no tokens were added
+to it.  The third screen shot uses the "removed" option, so it shows the conclusion a
+second time with the old text, with deletions marked in red.  Also, since the hypotheses are
+new, no line of old text is shown for them.
+
+.. _diffs_coqtop_on:
+
+  .. image:: ../_static/diffs-coqtop-on.png
+     :alt: coqtop with Set Diffs on
+
+.. _diffs_coqide_on:
+
+  .. image:: ../_static/diffs-coqide-on.png
+     :alt: |CoqIDE| with Set Diffs on
+
+.. _diffs_coqide_removed:
+
+  .. image:: ../_static/diffs-coqide-removed.png
+     :alt: |CoqIDE| with Set Diffs removed
+
+**How to enable diffs**
+
+.. opt:: Set Diffs %( "on" %| "off" %| "removed" %)
+.. This ref doesn't work: :opt:`Set Diffs %( "on" %| "off" %| "removed" %)`
```

Comment:
Issues with references:
- While the parameters (e.g. "on|"off"|"removed" should appear in the `opt ::` line, that seems to become part of the tag name.  I wanted line 581 to be a link to the option definition but couldn't get it to work with or without the parameters.
- There should be square brackets around the options instead of parentheses but I couldn't get that to work, either.  Known issue?

Reply:
Keep it with parentheses (like for https://coq.gitlab.io/-/coq/-/jobs/91412685/artifacts/_install_ci/share/doc/coq/sphinx/html/proof-engine/proof-handling.html#coq:opt.bullet-behavior-none-strict-subproofs).
It's very weird that the option doesn't appear in the index. @cpitclaudel Any insight?

Reply:
No particular idea